gnucash/import-export/import-main-matcher.cpp

```diff
@@ -689,6 +689,12 @@ gen_trans_common_toggled_cb (GtkCellRendererToggle *cell_renderer, gchar *path,
     else
         gnc_import_TransInfo_set_action (transaction_info, action);
     refresh_model_row (gui, model, &tree_iter, transaction_info);
+
+    auto treeview = GTK_TREE_VIEW(gui->view);
+    auto selection = gtk_tree_view_get_selection (treeview);
+    auto tree_path = gtk_tree_path_new_from_string (path);
+    gtk_tree_selection_select_path (selection, tree_path);
+    gtk_tree_path_free (tree_path);
```

Comment:
Let's make this a separate function `set_treeview_selection_from_path (GtkTreeView* view, const char* path)` and call it here for clarity.